.env.example

```diff
@@ -3,6 +3,9 @@
 # Replace the following with the output from "rake secret"
 APP_SECRET_TOKEN=REPLACE_ME_NOW!
 
+# Replace the following with another output from "rake secret"
+APP_ENCRYPTION_PASSPHRASE=REPLACE_ME_NOW!
```

Comment:
I wonder if we could make this easier to setup by defaulting it to the value of `APP_SECRET_TOKEN`, and saying "Huginn Credential values are encrypted by default using the value of the APP_SECRET_TOKEN. If you need to change the APP_SECRET_TOKEN, set APP_ENCRYPTION_PASSPHRASE to the old value so that secrets can still be decrypted." or something?